x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/googlevertexai/embeddings/GoogleVertexAiEmbeddingsServiceSettings.java

```diff
@@ -163,7 +163,6 @@ public String location() {
         return location;
     }
 
-    @Override
```

Comment:
nit: Maybe move this to the `projectId()` method

Reply:
Good point 👍 Added it to `projectId()`